### PR TITLE
Bump flytestdlib to v0.4.4

### PR DESCRIPTION
**Pull request by pingsutw**
Signed-off-by: Kevin Su <pingsutw@apache.org>

# TL;DR
update flytestdlib version to support docs generate command (`flytepropeller config docs`)
https://github.com/flyteorg/flytestdlib/releases/tag/v0.4.4

## Type
 - [ ] Bug Fix
 - [x] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
_Remove the '*fixes*' keyword if there will be multiple PRs to fix the linked issue_

## Follow-up issue
_NA_

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -17,7 +17,7 @@ linux_compile:
 .PHONY: compile
 compile:
 	mkdir -p ./bin
-	go build -o bin/flytepropeller ./cmd/controller/main.go
+	go build -o bin/flytepropeller ./cmd/controller/main.go && cp bin/flytepropeller ${GOPATH}/bin
```
Comment (EngHabu) [inline Makefile:20]: Let's revert this... if you want, you can add an `install` Makefile target... and if so it should do `go install`... let's not make assumptions about where the go bin will be...

**Review by EngHabu — APPROVED** [review]
